ktor-network/ktor-network-tls/jvm/src/io/ktor/network/tls/CipherSuitesJvm.kt

```diff
@@ -7,6 +7,6 @@ package io.ktor.network.tls
 import io.ktor.network.tls.platform.*
 
 internal actual fun CipherSuite.isSupported(): Boolean = when (platformVersion.major) {
-    "1.8.0" -> platformVersion.minor >= 161 || keyStrength <= 128
-    else -> keyStrength <= 128
+    "1.8.0" -> platformVersion.minor >= 161 || keyStrength <= 256
+    else -> keyStrength <= 256
```

Comment:
Probably we need to add other JDKs here instead.

Reply:
Hei @e5l , 

Can you help me point to correct resources to find out which jdk supports which cipher suite ? 

I looked at : https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#ciphersuites

and this supports all that are in the list : 

```
TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
TLS_RSA_WITH_AES_128_CBC_SHA
TLS_RSA_WITH_AES_128_GCM_SHA256
TLS_RSA_WITH_AES_256_CBC_SHA
```

Reply:
Hei @e5l, 

Can you help out here please ? I would love to get this into 1.5.2

Reply:
It depends on the actual JDK build. Right now the issue is that we should allow using all suites for JDK greater than 8

Reply:
Basically, we need to replace `else` with the condition on the `minor` version: (<=8) and allow any key for versions greater